.github/workflows/pr-test.yml

```diff
@@ -114,3 +114,12 @@ jobs:
         GIT_CONFIG_PARAMETERS: ${{ env.GGG_SMTP }}${{ env.GGG_REPO }}
       run: npm run test
       working-directory: gitgitgadget
+
+    - name: Clean up two day old branches and PRs
+      uses: actions/github-script@v3
+      id: clean-up
+      with:
+        script: |
+            const exports = require(`${process.env.GITHUB_WORKSPACE}/gitgitgadget/build/lib/delete-branches.js`)
```

Comment:
Maybe we can find a name that conveys that the code is only ever expected to be run during the integration tests?

Reply:
`delete-ci-test-branches`?  Sorry, my imagination is not running very wild.  The command line driver is not well named either.  It started as an inline script in the yaml but being able to run it from the command line or in the workflow made a lot more sense (especially for testing).

Wide open to naming suggestions.

Reply:
That name makes sense to me. Thanks!